src/mcMd/mdSimulation/MdSystem.cpp

```diff
@@ -135,10 +135,10 @@ namespace McMd
       setClassName("MdSystem");
 
       #ifndef INTER_NOPAIR
-      pairPotentialPtr_ = pairFactory().mdFactory(system.pairPotential());
-      if (pairPotentialPtr_ == 0) {
-         UTIL_THROW("Failed attempt to clone McPairPotential");
-      }
+      //pairPotentialPtr_ = pairFactory().mdFactory(system.pairPotential());
+      //if (pairPotentialPtr_ == 0) {
+      //   UTIL_THROW("Failed attempt to clone McPairPotential");
```

Comment:
I will probably add a more informative error message at lines 138:140 of the MdSystem.cpp constructor, to 
explain that constructing an MdSystem from an McSystem does not yet work for systems with a Coulomb
potential.  This comment is just a reminder for me to do this.